README.md

```diff
@@ -54,17 +54,29 @@ const fastify = require('fastify')()
 const { newAdapter } = require('casbin-pg-adapter').default
 const { newWatcher } = require('casbin-pg-watcher')
 
+
 const pgOptions = {
   connectionString: 'postgres://localhost',
   migrate: true
 }
 
-fastify.register(require('fastify-casbin'), {
-  model: 'basic_model.conf', // the model configuration
-  adapter: await newAdapter(pgOptions), // the adapter
-  watcher: await newWatcher(pgOptions) // the watcher
+const registerCasbin = async () => {
+  try {
+    await fastify.register(require('fastify-casbin'), {
+      model: 'basic_model.conf', // the model configuration
+      adapter: await newAdapter(pgOptions), // the adapter
+      watcher: await newWatcher(pgOptions) // the watcher
+    })
+  } catch (err) {
+    fastify.log.error(err)
+    process.exit(1)
+  }
+}
+registerCasbin().then(() => {
+  // Run the server after async registration
 })
```

Comment:
please don't change this, you're introducing an inconsistency with the previous example, and in generate this is not the recommended pattern to register plugins in fastify.